runtests.sh

```diff
@@ -13,8 +13,22 @@ echo "Running flake8"
 flake8 --show-source || { echo "flake8 errors found!"; exit 1; }
 
 echo "Running isort"
-isort --check-only --diff --quiet \
+isort \
+ --check-only \
+ --diff \
+ --quiet \
+ --skip __pycache__ \
+ --skip node_modules \
+ --skip migrations \
+ --skip-glob "*/.*/*" \
+ --skip-glob "*.md" \
+ --skip-glob "*.txt" \
+ --multi-line 1 \
+ --force-grid-wrap \
+ --lines 100 \
```

Comment:
`setup.cfg` is ignored, markdown and text files were "formatted"

Reply:
I started investigating this. When I fix it I will come edit this.

FYI I would be down for removing `isort` from here, adding it to Travis and rely on the `isort` from the `pre-commit` checks which is much faster. We would need to pass these arguments there as well.